mcr_monitoring/mcr_node_diagnostic/ros/src/mcr_node_analyzer.cpp

```diff
@@ -11,9 +11,8 @@
 
 using namespace std;
 
-PLUGINLIB_REGISTER_CLASS(mcr_node_analyzer,
-                         diagnostic_aggregator::mcr_node_analyzer,
-                         diagnostic_aggregator::Analyzer)
+PLUGINLIB_EXPORT_CLASS(diagnostic_aggregator::mcr_node_analyzer,
```

Comment:
This macro was missed by the migration script because it is split over two lines.